release-engineering/baseimage-exception-list.md

```diff
@@ -4,22 +4,69 @@ Theoretically, we expect to have all kubernetes images based on distroless (Plea
-| --------------------------------- | :---------------------------:|:----------------------:|
-|kube-proxy      |  [k8s.gcr.io/debian-iptable](https://github.com/kubernetes/kubernetes/blob/1b9d0c1094d31f851a5ec6e277fbf0b7382196cf/build/common.sh#L101)   | Requires iptable  |
-|etcd-empty-dir-cleanup | k8s.gcr.io/debian-base | Requires shell to do some cleanup |
-|etcd | k8s.gcr.io/debian-base | Requires shell to do data migration|
-|node-problem-detector| k8s.gcr.io/debian-base | Requires libsystemd0 |
-|fluentd-elasticsearch |[k8s.gcr.io/debian:stretch-slim](https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/fluentd-elasticsearch/fluentd-es-image/Dockerfile) |  Requires shell to install fluentd via ruby |
-|fluentd-gcp-scaler | [k8s.gcr.io/debian-base-amd64:v1.0.0](https://github.com/GoogleCloudPlatform/k8s-stackdriver/blob/master/fluentd-gcp-scaler/Dockerfile) |  Requires shell and kubectl |
-|stackdriver-logging-agent |  | requires fluentd |
-|ip-masq-agent | k8s.gcr.io/debian-iptables| Requires the iptables for all arches.|
-|kube-addon-manager | k8s.gcr.io/debian-base| Requires [shell](https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/addon-manager/kube-addons.sh).|
-|k8s-dns-kube-dns  |  k8s.gcr.io/debian-base  | Requires [container debugging tooling](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/20190212-ephemeral-containers.md) |
-|k8s-dns-sidecar  |  k8s.gcr.io/debian-base  |  Requires [container debugging tooling](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/20190212-ephemeral-containers.md) |
-|k8s-dns-dnsmasq-nanny  |  k8s.gcr.io/debian-base  |  Requires [container debugging tooling](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/20190212-ephemeral-containers.md) |
-|k8s-dns-node-cache  |  k8s.gcr.io/debian-base  |  Requires [container debugging tooling](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/20190212-ephemeral-containers.md) |
-|kube-apiserver | k8s.gcr.io/debian-base| Causes performance regression, requires further investigation on klog when running 2000 nodes scalability test. OWNER:@yuwenma|
-|kube-controller-manager | k8s.gcr.io/debian-base| Causes performance regression, requires further investigation on klog when running 2000 nodes scalability test. OWNER:@yuwenma|
-|kube-scheduler | k8s.gcr.io/debian-base| Causes performance regression, requires further investigation on klog when running 2000 nodes scalability test. OWNER:@yuwenma|
+## Release images
+
+### Release: `debian-base`
+
+| Image Name | Reasons for exception |
+|---|---|
+| [debian-iptables] | Needed to supported images that require `iptables` |
+| [kube-controller-manager][core-images] | `klog` 2000 node scalability regressions, [Flexvolume master driver is blocking controller-manager from moving to distroless](https://github.com/kubernetes/kubernetes/issues/78737) |
```

Comment:
Please remove the `"klog" 2000 node scalability regressions,` , no longer applicable

Reply:
@dims -- Fixed!